test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -18,37 +18,26 @@ class test_ucp_ep_reconfigure : public ucp_test {
     using address_t = std::pair<void*, ucp_unpacked_address_t>;
     using address_p = std::unique_ptr<address_t, void (*)(address_t*)>;
 
-    class entity {
+    class entity : public ucp_test_base::entity {
     public:
-        void connect(const entity &other, bool is_exclude_iface);
-
-        void verify_configuration(const entity &other) const;
-
-        bool is_reconfigured() const
+        entity(const ucp_test_param &test_param, ucp_config_t *ucp_config,
+               const ucp_worker_params_t &worker_params,
+               const ucp_test_base *test_owner, bool exclude_ifaces) :
+            ucp_test_base::entity(test_param, ucp_config, worker_params,

```

Comment:
i'm not sure we need the prefix ucp_test_base:: in this context (multiple locations)

Reply:
compiler error is generated without it.
How should it be distinguished from reconfig::entity?

Reply:
right